MAUI/TextInputLayout/Events.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
```

Comment:
Why tags are modified? The trademark symbol is improper.

Reply:
I am already facing MD tag validation errors. There seems to be a mistake in the highlight tag on line number 28.
So I fix it by changing the tags